geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/JdbcClusterConfigDistributedTest.java

```diff
@@ -51,8 +51,6 @@ public static void beforeClass() {
   @Test
   public void recreateCacheFromClusterConfig() throws Exception {
     gfsh.connectAndVerify(locator);
-    gfsh.executeAndAssertThat("create jdbc-connection --name=connection --url=url")
-        .statusIsSuccess();
     gfsh.executeAndAssertThat(
         "create jdbc-mapping --region=regionName --connection=connection --table=testTable --pdx-class-name=myPdxClass --value-contains-primary-key --field-mapping=field1:column1,field2:column2")
```

Comment:
What is "--connection" is referring to here? Is it supposed to be jndi binding name?

Reply:
It is the jndi-binding's name. We have another story to figure out how we want to refer to the "jndi-binding". Once we do, then in that story we will convert the various things like this to use that name.